<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NASA APOD Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>830</width>
      <height>266</height>
     </rect>
    </property>
    <property name="title">
     <string>NASA APOD</string>
    </property>
    <widget class="QLabel" name="lbl_explanation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>576</width>
       <height>225</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_thumbnail">
     <property name="geometry">
      <rect>
       <x>604</x>
       <y>30</y>
       <width>225</width>
       <height>225</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Click to see Full Photo</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>285</y>
      <width>830</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="toolTip">
        <string>Pick a date to see an APOD</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_display_apod">
       <property name="toolTip">
        <string>Get NASA Astronomy Photo of the Day</string>
       </property>
       <property name="text">
        <string>Get APOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_full_photo">
       <property name="toolTip">
        <string>Display full photo</string>
       </property>
       <property name="text">
        <string>Full Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_hd_photo">
       <property name="toolTip">
        <string>Display HD photo</string>
       </property>
       <property name="text">
        <string>HD Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_random_photo">
       <property name="toolTip">
        <string>Display HD photo</string>
       </property>
       <property name="text">
        <string>Random Photo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_file">
       <property name="text">
        <string>&amp;Save Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
